src/seedu/addressbook/Main.java

```diff
@@ -108,10 +108,17 @@ private void recordResult(CommandResult result) {
     private CommandResult executeCommand(Command command)  {
         try {
             command.setData(addressBook, lastShownList);
-            CommandResult result = command.execute();
+            CommandResult result = command.execute(); //start work
             storage.save(addressBook);
             return result;
-        } catch (Exception e) {
+
+        //Added this to print a error message where the user accidentally makes the storage file read only
```

Comment:
This style of comments is not consistent with the rest of the codebase, i.e., the comment should not sound like a justification.